app/helpers/audio_visual_helper.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+#
+# Helper methods to generate the s3 presigned URL for audio/video files
+module AudioVisualHelper
+  # @param [String] s3 key
+  # @return [String] presigned URL
+  # @note For development environments, we need to subsitute the service hostname of S3 ('minio')
+  #       with 'localhost' for links to resolve. In production, 'AWS_ENDPOINT_URL's hostname is valid.
+  def s3_url(key)
+    if ENV['AWS_ENDPOINT_URL'].blank?
+      Rails.logger.warn('AWS_ENDPOINT_URL environment variable is not defined.')
+      return ""
+    end
+    client_opts = {}
+    client_opts = { endpoint: ENV['AWS_ENDPOINT_URL']&.sub('minio', 'localhost') } if Rails.env.development?
```

Comment:
try pulling out the `&` after `ENV['AWS_ENDPOINT_URL']` to fix coverage? since you're already testing to see if it's empty above, you don't need to assume it might not be there at this point